amen/audio.py

```diff
@@ -18,4 +20,26 @@ def __init__(self, file_path, convert_to_mono=False, sample_rate=22050):
         self.raw_samples = y
         self.num_channels = y.ndim
         self.duration = librosa.get_duration(y=y, sr=sr)
+        self.features = self.create_features()
+
+    def create_features(self):
+        features = {}
+        features['centroid'] = self.get_centroid()
+        return features
+
+    def get_centroid(self):
+        """
+        Gets spectral centroid data from librosa and loads it into a feature
+        """
+        hop_length = 512
+        mono_samples = librosa.to_mono(self.raw_samples)
+        centroid = librosa.feature.spectral_centroid(mono_samples, hop_length=hop_length)
```

Comment:
I think we should try to consolidate dsp parameters in one central location, rather than defining them locally within each feature function.  (To whatever extent that's possible.)

Side note: I just put out a [module](https://pypi.python.org/pypi/presets) to make this sort of thing much easier.